calendar.php

```diff
@@ -140,11 +140,20 @@
                                 if (isset($events[$e])) {
                                     $dayEvents = $events[$e];
                                     foreach ($dayEvents as $info) {
-                                        if($info["completed"] == "no"){
-                                            $eventsStr .= '<a class="calendar-event" style="background-color:#1a7024" href="event.php?id=' . $info['id'] . '">' . $info['name'] .  '</a>';
+                                        if($info['restricted_signup'] == 1) { //Restricted Event
+                                            $backgroundCol = '#c73d06'; // Sets background color to a red to show the restriction
```

Comment:
Good idea to use color to differentiate between restricted and open events. Maybe we could consider doing something similar to indicate events that the user is signed up for?

Reply:
Oh yeah that's a good point, maybe the green is for a standard event, red is for restricted, and then maybe we can use the standard StepVA blue for events they have signed up for just for increased contrast